src/registrar/tests/test_views.py

```diff
@@ -1415,11 +1415,34 @@ def test_domain_invitation_email_sent(self):
             add_page.form["email"] = email_address
             self.app.set_cookie(settings.SESSION_COOKIE_NAME, session_id)
             add_page.form.submit()
+
+        expected_email_content = {
```

Comment:
**comment (blocking)**: This does not test the changes that you made, and it is fragile because this test will fail if we change anything about the email. I think that you should change the mock of the `send_email` method so that you can make assertions about the `Content` of the sent message. Then you should set up all of the different cases that your code works over: requester as request.user, requester as creator, and empty names so it uses emails for both. 

If you manually tested this code to make sure that it did the right thing, then you should turn each of those manual tests into unit tests so that we can verify on an ongoing basis that this code is doing the right thing.

Reply:
Will do, thanks